tests/behat/features/capabilities/group/group-create-secret.feature

```diff
@@ -0,0 +1,182 @@
+@api @group @stability @stability-1 @group-create-secret
+Feature: Create Secret Group
+  Benefit: I want to create a secret group, where only group members can see the content.
+  Role: As a LU
+  Goal/desire: I want to create Secret Groups
+
+  Scenario: Successfully create secret group
+    Given users:
+      | name             | mail                     | status | roles       |
+      |  SecretGroup User One   | group_user_1@example.com | 1      | sitemanager |
+      |  SecretGroup User Two   | group_user_2@example.com | 1      |             |
+    And I enable the module "social_group_secret"
+    And I am logged in as an "authenticated user"
+    And I am on "user"
+    And I click "Groups"
+    And I click "Add a group"
+    Then I should not see "Secret group"
+    Given I am logged in as "SecretGroup User One"
+    And I am on "user"
+    And I click "Groups"
+    And I click "Add group"
+    Then I click radio button "Secret group This is a secret group. Users can only join by invitation and the group itself and its content are hidden from non members." with the id "edit-group-type-secret-group"
+    And I press "Continue"
+    When I fill in "Title" with "Test secret group"
+    And I fill in the "edit-field-group-description-0-value" WYSIWYG editor with "Description text"
+    And I fill in "Location name" with "Disclosed"
+    And I select "NL" from "Country"
+    And I wait for AJAX to finish
+    Then I should see "City"
+    And I fill in the following:
+      | City | Hengelo |
+      | Street address | Padangstraat 11 |
+      | Postal code | 7556SP |
+    And I press "Save"
+    And I should see "Test secret group" in the "Main content"
+    And I should see "Disclosed"
+    And I should see "1 member"
+    And I should see "Joined"
+    And I should not see the link "Read more"
+
+    And I should see "Test secret group" in the "Hero block"
+    And I should see the button "Joined"
+    And I should see the link "Edit group" in the "Hero block"
+    And I should see "Disclosed" in the "Hero block"
+    And I should see "Padangstraat 11" in the "Hero block"
+    And I should see "7556SP" in the "Hero block"
+
+  # Create a post inside the secret group, visible to group members only
+    When I click "Stream"
+    And I fill in "Say something to the group" with "This is a secret group post."
+    And I select post visibility "Group members"
+    And I press "Post"
+    Then I should see the success message "Your post has been posted."
+    And I should see "This is a secret group post."
+
+  # Create a topic inside the secret group
```

Comment:
I think we can make this a separate scenario, starting with "Given secret group...."

We also don't actually test the visibility here but only that creation works.